ui/imports/shared/controls/WalletAccountListItem.qml

```diff
@@ -0,0 +1,79 @@
+import QtQuick 2.15
+
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Utils 0.1 as StatusQUtils
+
+import AppLayouts.Wallet 1.0
+
+import utils 1.0
+
+StatusListItem {
+    id: root
+
+    property var modelData
+    property string chainShortNames
+    property bool clearVisible: false
+    signal cleared()
+
+    objectName: !!modelData  ? modelData.name: ""
+
+    height: visible ? 64 : 0
+    title: !!modelData && !!modelData.name ? modelData.name : ""
+    subTitle:{
+        if(!!modelData) {
+            let elidedAddress = StatusQUtils.Utils.elideText(modelData.address,6,4)
+            return sensor.containsMouse ? WalletUtils.colorizedChainPrefix(chainShortNames) + Utils.richColorText(elidedAddress, Theme.palette.directColor1): chainShortNames + elidedAddress
+        }
+    }
+    statusListItemSubTitle.wrapMode: Text.NoWrap
+    asset.emoji: !!modelData && !!modelData.emoji ? modelData.emoji: ""
+    asset.color: !!modelData ? modelData.color: ""
+    asset.name: !!modelData && !modelData.emoji ? "filled-account": ""
+    asset.letterSize: 14
+    asset.isLetterIdenticon: !!modelData && !!modelData.emoji ? true : false
+    asset.bgColor: Theme.palette.indirectColor1
+    radius: 0
+    color: sensor.containsMouse || highlighted ? Theme.palette.statusListItem.hightlightColor : "transparent"
+    components: [
+        Column {
+            anchors.verticalCenter: parent.verticalCenter
+            StatusTextWithLoadingState   {
+                anchors.right: parent.right
+                font.pixelSize: 15
+                text: LocaleUtils.currencyAmountToLocaleString(!!modelData ? modelData.currencyBalance: "")
+            }
+            StatusIcon {
+                anchors.right: parent.right
+                width: 15
+                height: !!icon ? 15 : 0
+                color: Theme.palette.directColor1
+                icon: {
+                    if(!!modelData) {
+                        if (modelData.walletType === Constants.watchWalletType)
+                            return "show"
+                        if (modelData.walletType === Constants.keyWalletType)
+                            return "keycard"
```

Comment:
@Khushboo-dev-cpp it should not be like this.  `keyWalletType` refers to accounts imported by private key, not to a keycard accoutns.

Reply:
aaaah , didnt know, how do I know if an account is on the keycard?

Reply:
ok I found it , it is migratedToKeycard variable, I'll use that :)

I actually used this code from the Left side accounts list and didnt think it might be wrong, will update there as well

Reply:
> I actually used this code from the Left side accounts list and didnt think it might be wrong, will update there as well

I've just checked and yes, it's wrong there as well. Since that's a single line change, if you don't mind, please change it there as well.

Reply:
sure , its already done